packages/expo-device/ios/UIDevice.swift

```diff
@@ -156,10 +154,10 @@ public extension UIDevice {
         return ExpoDeviceType(modelName: "iPad Pro (12.9-inch) (5th generation)", deviceYearClass: 2021)
       case "iPad14,5-A", "iPad14,5-B", "iPad14,6-A", "iPad14,6-B":
         return ExpoDeviceType(modelName: "iPad Pro (12.9-inch) (6th generation)", deviceYearClass: 2022)
-      case "AppleTV5,3":
```

Comment:
this is removed because these are already down in the `if os(tvOS)` directive (this was actually a duplicate and within the wrong directive (`if os(iOS)`)